cabal.project

```diff
@@ -1,3 +1 @@
 packages: .
-
-with-compiler: ghc-8.8.3
```

Comment:
I’m not sure that this is the right thing to have in a repo-level cabal.project, but let me know if there’s a reason why it’s this way.